spec/features/email_spec.rb

```diff
@@ -32,4 +32,16 @@
       expect(current_email).to have_content 'Szanowna Pani/ Szanowny Panie'
     end
   end
+    scenario 'sprawdzenie dodania hasła' do
+	clear_emails
+
+	visit '/submissions'
+	click_on 'Alicja w krainie czarów'
```

Comment:
Nie wystarczyłoby przenieść drugie end z poprzedniego 'scenario' za mój? Bo
nie do końca rozumiem, myślałam, że /submission to strona, na której dodaje
nowe zgłoszenie.

W dniu 24 lutego 2016 12:54 użytkownik Aleksander Smywiński-Pohl <
notifications@github.com> napisał:

> In spec/features/email_spec.rb
> <https://github.com/apohllo/rocznik/pull/353#discussion_r53928204>:
>
> > @@ -32,4 +32,16 @@
> >        expect(current_email).to have_content 'Szanowna Pani/ Szanowny Panie'
> >      end
> >    end
> > +    scenario 'sprawdzenie dodania hasła' do
> > +	clear_emails
> > +
> > +	visit '/submissions'
> > +	click_on 'Alicja w krainie czarów'
>
> Nie ma takiego artykułu (Submission) w bazie danych. Trzeba go dodać w
> bloku before proszę popatrzeć na inne testy.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apohllo/rocznik/pull/353/files#r53928204>.
>